api/doc/v1/CHANGELOG.md

```diff
@@ -0,0 +1,69 @@
+# Aptos Node API Changelog
+
+All notable changes to the Aptos Node API will be captured in this file. This changelog is written by hand for now. It adheres to the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+**Note**: The Aptos Node API does not follow semantic version while we are in active development. Instead, breaking changes will be announced with each devnet cut. Once we launch our mainnet, the API will follow semantic versioning closely.
+
+## 1.0.0 (2022-08-04)
```

Comment:
The only thing I ask in date formatting is to explicitly say `(04 August 2022)` which 100% avoids any ambiguity. Even after so many years in the good old USA, I still can't get used the idea of month first, for example.

Reply:
I'm a pretty firm believer in YYYY-MM-DD, I don't think anyone would confuse this for YYYY-DD-MM.

Reply:
Haha, okay then.

Reply:
Spoke offline, we'll keep YYYY-MM-DD.